plugins/Live/templates/_totalVisitors.twig

```diff
@@ -16,13 +16,13 @@
         <tbody>
         <tr class="">
             <td class="label column">{{ 'Live_LastHours'|translate(24) }}</td>
-            <td class="column">{{ visitorsCountToday }}</td>
-            <td class="column">{{ pisToday }}</td>
+            <td class="column">{{ visitorsCountToday|number }}</td>
+            <td class="column">{{ pisToday|number }}</td>
```

Comment:
Haven't tested your changes yet, but isn't the formatting done twice this way.
The `number` filter for twig uses [this method](https://github.com/piwik/piwik/blob/3.x-dev/core/Twig.php#L36-L40), which calls `NumberFormatter::format`, but that is already done in Controller. So guess it should be enough to do it either in Controller or in View.

Reply:
@sgiehl - Makes sense,  missed the twig formatting referencing that function. I'll test locally and push up some changes that only has the formatting in the view so that all of the items are calling the same helper function.